test/html/multiple-imports.html

```diff
@@ -7,5 +7,6 @@
     Code distributed by Google as part of the polymer project is also
     subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
+<link rel="import" href="multiple-imports.html">
```

Comment:
Ah I think I understand now. Yea, I'd definitely like to see a clean test fixture for this case. New test fixtures are cheap, and I worry about polluting an already-in-use test with something as strange as this.